src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5515,7 +5515,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>The type name '{0}' is reserved to be used by the compiler.</value>
   </data>
   <data name="ERR_InExtensionMustBeValueType" xml:space="preserve">
-    <value>The first parameter of an 'in' extension method '{0}' must be a value type.</value>
+    <value>The first parameter of an 'in' extension method '{0}' must be a concrete (non-generic) value type.</value>
```

Comment:
The word "an" I believe should be "the", since the error message refers to a specific method by name.

Reply:
Done